privacyidea/lib/twostep.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+#  2017-11-13   Friedrich Weber <friedrich.weber@netknights.it>
+#               Add a module containing general twostep initialization utils
```

Comment:
Do you expect more to come?

Otherwise we could add this to the tokenclass.py as 

DEFAULT_TWOSTEP_CLIENTSIZE = 8...

Reply:
815577b943a9090ec4211bfac7bf8655130c3a31 moves these constants to hotptoken.py ... Is that ok, or should I move them to tokenclass.py?